src/components/title-with-description/TitleWithDescription.tsx

```diff
@@ -15,12 +14,14 @@ interface TitleWithDescriptionProps {
     title?: SxProps
     description?: SxProps
   }
+  useTooltip?: boolean
 }
 
 const TitleWithDescription = ({
   title,
   description,
-  style = styles
+  style = styles,
+  useTooltip = false
```

Comment:
Maybe **withDescriptionTooltip** or **isDescriptionTooltip** name will be more clear

Reply:
fixed